garage/init-garage.sh

```
@@ -1,65 +1,63 @@
+    # 2. Create API-key
+    echo "--- 🔑 Create admin-key ---"
     garage key create my-admin-key || true
     KEY_ID=$(garage key list | grep "my-admin-key" | awk '{print $1}')
     echo "S3 Access Key ID: $KEY_ID"
 
-    # 3. Skapa Bucket
-    echo "--- 📦 Skapar bucket ---"
-    garage bucket create my-bucket || true
+    # 3. Create Bucket
+    echo "--- 📦 Create bucket ---"
+    garage bucket create ${S3_BUCKET_NAME} || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)